### PR TITLE
Add initial content for resolving ELN build failures.

### PR DESCRIPTION
**Pull request by mmathesius**
I'm looking for feedback and/or suggestions for additional tips!

You may find it easier to view by letting github render it at https://github.com/mmathesius/eln-docs/blob/ftbfs-landing-page/modules/ROOT/pages/ftbfs.adoc

### CODE REVIEW

File: modules/ROOT/pages/ftbfs.adoc [hunk 1]
```diff
@@ -1,31 +1,123 @@
...
+
+```
+%if 0%{?fedora} >= 29 || 0%{?rhel} >= 9
+BuildRequires: SomePackage
+%endif
+```
+
+This conditional will be met when `%\{rhel}` is defined as 9 or greater.
+It is also shareable between earlier and later versions of Fedora and RHEL.
+Even better is using `<` to limit behaviors to RHEL 8 and earlier:
```
Comment (tdawson) [inline modules/ROOT/pages/ftbfs.adoc:66]: I don't think we should globally say it's better.  It depends on the circumstances.  What if we changed it from
  Even better is using
to
  Often it is better to use

**Comment by tdawson** [comment]
Other than my one comment above, I really like it.

File: modules/ROOT/pages/ftbfs.adoc (hunk 1 above)
Comment (mmathesius) [reply modules/ROOT/pages/ftbfs.adoc]: I agree. I just pushed another commit incorporating that, as well as adding a short section about handling the recurring python 2 deprecation failures. (Although most of those have probably already been fixed.)

Comment (tdawson) [reply modules/ROOT/pages/ftbfs.adoc]: Update looks good.
I like the call out to Python2.

**Comment by mmathesius** [comment]
Commits squashed and `[WIP]` removed. Thanks for the reviews!

**Pull request merged**